gwnr/waveform/esigma_utils.py

```diff
@@ -505,7 +511,8 @@ def get_imr_esigma_modes(
 
     # Retrieve modes, orbital phase and frequency from the returned list
     orb_eccentricity = retval[-2]["e"]
-    orb_freq = retval[-2]["phidot"] / ((mass1 + mass2) * lal.MTSUN_SI) / (2 * np.pi)
+    if (f_window_mr_transition is None) or failsafe or (verbose > 1):
```

Comment:
With this, now we will have to be very careful in adding new prints or code that use `orb_freq` below. Just making a note of this here.